src/mol-model/structure/structure/util/superposition-sifts-mapping.ts

```diff
@@ -69,25 +69,39 @@ function getPositionTables(index: IndexEntry[], pivot: number, other: number, N:
 
         const l = Math.min(a[2] - a[1], b[2] - b[1]);
 
-        // TODO: allow to use just backbone atoms?
         // TODO: check if residue types match?
         for (let i = 0; i < l; i++) {
-            let eI = (a[1] + i) as ElementIndex;
+            const eI = (a[1] + i) as ElementIndex;
+            const eJ = (b[1] + i) as ElementIndex;
+            if (traceOnly && (!traceAtom(a[0], eI) || !traceAtom(b[0], eJ))) continue;
```

Comment:
This will only work if the atoms are in the same order, which might not be the case. I know the previous code didn't handle that either, but I can complain about that here :D

Need to have something like:

```ts
if (traceOnly) {
  const  trace = findTraceAtoms(a, a[1], a[2]);
  for (let j = b[1]; j < b[2]; j++) {
    if (!trace.has(...)) continue;
    ...
  }
}
```

Reply:
Reverted back to some of the original code and added an optional step that narrows the range down to the backbone atom if traceOnly (b21a78a). Seems to work on a manipulated file.

The `BB` check might be relevant when dealing with a SIFTS mapping, correct?